pages/linux/pacman4console.md

```diff
@@ -0,0 +1,20 @@
+# pacman4console
+
+> A text-based console game inspired by the original Pacman.
+> You play as Pacman (C); avoid the ghosts (&'s); collect all the pellets (*'s and .'s).
```

Comment:
Perhaps this will read better? Also, perhaps we could add the repository url too?
```suggestion
> You play as Pacman (the `C`), avoid the ghosts (the `&`s), and collect all the pellets (the `*`s and `.`s).
> More information: <https://github.com/YoctoForBeaglebone/pacman4console>.
```